grunt-configs/shell.js

```diff
@@ -31,6 +31,34 @@ module.exports = function(grunt, options) {
                     cwd: 'facia-tool/public'
                 }
             }
+        },
+        jspmBundleFaciaToolCollections: {
+            command: 'node ../../node_modules/jspm/jspm.js bundle --minify ' + [
```

Comment:
Interested in why your using `bundle` instead of `bundle-sfx` to avoid two HTTP requests?